content/references/http-websocket-apis/api-conventions/request-formatting.md

```diff
@@ -98,11 +98,15 @@ The commandline always uses the latest [API version](#api-versioning).
 
 **Caution:** The commandline interface is intended for administrative purposes only and is _not a supported API_. New versions of `rippled` may introduce breaking changes to the commandline API without warning!
 
+
 ## API Versioning
 
-The `rippled` server uses a single integer to identify the API version to use. The first API version is `1`; currently, this is the only version of the `rippled` API. (There is no API version 0.) [New in: rippled 1.5.0][]
+The `rippled` server uses a single integer to identify the API version to use. Currently, there are two API versions: `1` and `2` [New in: rippled 2.0.0][]. Servers support a range of API versions, which it reports in the `version` API method. <!-- STYLE_OVERRIDE: will --> <!-- TODO: add a link when `version` method is documented. -->
+
+Separate API requests can use different API versions even on the same persistent connection. For example, if you connect WebSocket to a server that supports API versions 1 and 2, you can make a `server_info` request using API version 2 and then make another `server_info` request using API version 1 from the same connection.
```

Comment:
I think `server_info` may be unchanged in version 2, so I'm not sure this is the best example.